app/views/common/_menu.html.haml

```diff
@@ -1,14 +1,22 @@
-.row
-  .col-md-10
-    %ul.nav.nav-pills
-      %li
-        = link_to 'Главная', root_path
-      %li
-        = link_to 'Список хранилищ', storages_path
-      %li
-        = link_to 'Новое хранилище', new_storage_path
-  .col-md-2
-    .pull-left
-      = "Вы вошли как: <b>#{current_user.full_name}</B>".html_safe
-    .pull-right
-      = link_to 'Выйти', destroy_user_session_path, method: :delete
\ No newline at end of file
+%nav.navbar.navbar-default.navbar-static-top
+  .container
+    .row
+      .col-md-9
+        %ul.nav.navbar-nav
+          %li
+            = link_to 'Главная', root_path
+          %li
+            = link_to 'Список хранилищ', storages_path
+          %li
+            = link_to 'Новое хранилище', new_storage_path
+      .col-md-3
+        .dropdown
+          %button.btn.btn-default.dropdown-toggle{'type' => 'button', 'id' => 'dropdownMenu1', 'data-toggle' => 'dropdown', 'aria-haspopup' => 'true', 'aria-expanded' => 'true'}
+            = current_user.full_name
+            %span.caret
+          %ul.dropdown-menu{'aria-labelledby' => 'dropdownMenu3'}
+            %li
+              = link_to 'Настройки', edit_user_path(current_user)
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace